Gemfile

```diff
@@ -26,6 +26,7 @@ gem 'config', '~> 2.0'
 gem 'dor-services-client', '~> 12.0'
 gem 'dor-workflow-client', '~> 5.0'
 gem 'druid-tools'
+gem 'globus_client'
```

Comment:
I was wondering about whether we should pin this or not, but I don't think we need to sweat that now.

Reply:
Is the globus client used?

Reply:
@justinlittman it is not. It was something we discussed keeping here or including when necessary. I'm happy to drop it to avoid any confusion.